students/tbrackney/lesson4/mailroom2.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python3
+"""
+File Name: mailroom2.py
+Author: Travis Brackney
+Class: Python 201 - Self paced online
+Date Created 4/22/2018
+Python Version: 3.6.4
+"""
+
+
+''' Populating initial donors list'''
+# key is donor name, values are list of donations
+donors = {}
+donors['Tom Selleck'] = [2000.00, 1500.00, 500.00]
+donors['Burt Reynolds'] = [45.00]
+donors['Nick Offerman'] = [1000.00, 1000.00]
+donors['Sam Elliot'] = [1200.00, 550.00]
+donors['John Waters'] = [20.00, 20.00, 20.00]
+
+
+# updated
+def list_donors():
+    ''' Prints list of donors '''
+    for d in donors:
+        print(d)
+    return
+
+
+# Updated
+def add_donor(d_name):
+    '''Adds a new donor to donors'''
+    donors[d_name] = []
+    add_donation(d_name)
+    return
+
+
+# Updated
+def add_donation(d_name):
+    '''Appends a donation to existing donor '''
```

Comment:
note that docstrings use triple double quotes -> `"""Appends a donation to existing donor"""`